docs/docs/parachains/picasso/tokenomics.md

```diff
@@ -0,0 +1,27 @@
+|:-------------------------------------------------------:|---------------:|:----------------:|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|  Founders, Early Team Members and Current Contributors  |  1,365,000,000 |      13,65%      | 13.65% of PICA are earmarked for distribution to founders, early team members, and current key team members along a 2-year linear vesting schedule with an initial 6-month lock.                                                                                                                                                                                                                                                       |
+|                 Key Partners & Advisors                 |    635,000,000 |      6,35%       | 6.35% of PICA are intended to be distributed to key strategic partners and advisors along a 2-year linear vesting schedule with a 6 month lock. This will form part of the Team allocation.                                                                                                                                                                                                                                            |
+|                        Treasury                         |  1,800,297,200 |       ~18%       | Approximately 18% of PICA’s supply is allocated to account for the long-term sustainability of the Picasso Network. Picasso Network treasury receives transaction fees and is governed by the Picasso General Council, which is intended to later be governed by PICA token holders. 75% of spent network fees will automatically flow into this fund (the remaining 25% of spent fees are intended to be distributed to collators).** |
+|               Future Employee Incentives                |    200,000,000 |        2%        | 2% of PICA is intended to be reserved for future employee incentives. Future issues from this pool will be subject to a 4-year vesting schedule from network launch or grant date (whichever is later) with a 1-year cliff and monthly vesting thereafter.**                                                                                                                                                                           |
```

Comment:
the 2 and 16 are supposed to be subcategories of the 18%

Reply:
Best I can for this in markdown short term is put something like "Future employee incentives" (as part of Treasury)
Additional Allocations (as part of Treasury)